crates/futures/src/lib.rs

```diff
@@ -101,12 +101,21 @@
 //! }
 //! ```
 
+#![feature(stdsimd)]
 #![deny(missing_docs)]
 
 #[cfg(feature = "futures_0_3")]
```

Comment:
While you're at it it's probably good to remove this if `"atomics"` is enabled (since it's not compatible).

The `cfg-if` crate may be able to help clean up the `#[cfg]` here a bit too.